src/common/mailing.ts

```diff
@@ -97,12 +97,20 @@ interface EmailContact {
   custom_fields: Record<string, string>;
 }
 
-const profileToContact = (profile: User, contactId: string) => {
-  const genericInviteURL = getInviteLink({
+const profileToContact = async (
+  profile: User,
+  contactId: string,
+  log: FastifyBaseLogger,
+) => {
+  const rawInviteURL = getInviteLink({
     referralOrigin: 'generic',
     userId: profile.id,
   });
 
+  const [genericInviteURL] = await Promise.all([
+    getShortUrl(rawInviteURL.toString(), log),
+  ]);
```

Comment:
Why promise.all here?

Reply:
Ah I actually had 2 items there before, let me adjust it :)